src/reducers/alerts.js

```diff
@@ -2,29 +2,47 @@ import alertsData from '../lib/alerts/index.jsx';
 import {AlertTypes, AlertLevels} from '../lib/alerts/index.jsx';
 import extensionData from '../lib/libraries/extensions/index.jsx';
 
-const SHOW_STANDARD_ALERT = 'scratch-gui/alerts/SHOW_STANDARD_ALERT';
+const SHOW_ALERT = 'scratch-gui/alerts/SHOW_ALERT';
 const SHOW_EXTENSION_ALERT = 'scratch-gui/alerts/SHOW_EXTENSION_ALERT';
 const CLOSE_ALERT = 'scratch-gui/alerts/CLOSE_ALERT';
 const CLOSE_ALERTS_WITH_ID = 'scratch-gui/alerts/CLOSE_ALERTS_WITH_ID';
 
+/**
+ * Initial state of alerts reducer
+ *
+ * {bool} visible - whether the alerts are visible
+ * {array} alertsList - list of alerts, each with properties:
+ *  * alert type (required): one of AlertTypes
+ *  * closeButton (optional): bool indicating that we should show close button
+ *  * content (optional): react element (a <FormattedMessage />)
+ *  * extentionId (optional): id string that identifies the extension
+ *  * iconURL (optional): string
+ *  * level (required): string, one of AlertLevels
+ *  * message (optional): string
+ *  * showReconnect (optional): bool
+ */
 const initialState = {
     visible: true,
-    // list of alerts, each with properties:
-    // * alert type (required): one of AlertTypes
-    // * closeButton (optional): bool indicating that we should show close button
-    // * content (optional): react element (a <FormattedMessage />)
-    // * extentionId (optional): id string that identifies the extension
-    // * iconURL (optional): string
-    // * level (required): string, one of AlertLevels
-    // * message (optional): string
-    // * showReconnect (optional): bool
     alertsList: []
 };
 
+const filterPopupAlerts = alertsList => (
+    alertsList.filter(curAlert => (
+        curAlert.alertType === AlertTypes.STANDARD ||
+        curAlert.alertType === AlertTypes.EXTENSION
+    ))
+);
+
+const filterInlineAlerts = alertsList => (
+    alertsList.filter(curAlert => (
+        curAlert.alertType === AlertTypes.INLINE
+    ))
+);
+
 const reducer = function (state, action) {
     if (typeof state === 'undefined') state = initialState;
     switch (action.type) {
-    case SHOW_STANDARD_ALERT: { // also will show inline alerts
+    case SHOW_ALERT: { // shows standard and inline alerts
```

Comment:
To me this comment implies that `SHOW_ALERT` won't show extension alerts, but that doesn't look like it's true.  This comment doesn't really clarify anything, so I think it could be removed.

```suggestion
    case SHOW_ALERT: {
```

Reply:
Sorry, I'm confused -- `SHOW_ALERT` certainly isn't intended to show extension alerts. It's true that it does not explicitly check for them, but neither will it show them correctly, if you tried! Is that what you mean?

Reply:
Yeah it's just a little misleading.

Reply:
I still don't totally undertand your point, but I changed the comment to `// intended to show standard and inline alerts, but not extensions` which I think is clearer